articles/contributing/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Contributing to the Quantum Development Kit | Microsoft Docs
+description: Contributing to the Quantum Development Kit
+author: cgranade
+ms.author: chgranad
+ms.date: 10/12/2018
+ms.topic: article
+uid: microsoft.quantum.contributing
+---
+
+# Contributing to the Quantum Development Kit #
+
+The Quantum Development Kit is more than a collection of tools for writing quantum programs.
+It's part of a broad community of people discovering quantum computing, performing research in quantum algorithms, developing new applications for quantum devices, and otherwise working to make the most out of the quantum programming.
```

Comment:
Software (or hardware) is typically not considered part of communities of people.